File: data/imported/knowledge/slides/java/merged/snippets/snippet_077.java
@Test
    void testInsertAutoShape() {
        try (var pres = new Presentation()) {
            var slide = blankSlide(pres);
            slide.getShapes().addAutoShape(ShapeType.RECTANGLE, 50, 50, 200, 100);
            slide.getShapes().addAutoShape(ShapeType.ELLIPSE, 300, 50, 150, 150);
            slide.getShapes().insertAutoShape(1, ShapeType.TRIANGLE, 150, 200, 100, 100);
            assertThat(slide.getShapes().size()).isEqualTo(3);
        }
    }

File: data/imported/knowledge/slides/net/merged/snippets/snippet_077.cs
[Fact]
    public void TestClearShapes()
    {
        // clear() empties the shape collection.
        using var pres = new Presentation();
        var slide = pres.Slides[0];
        slide.Shapes!.AddAutoShape(ShapeType.Rectangle, 50, 50, 200, 100);
        slide.Shapes!.Clear();
        slide.Shapes!.Count.Should().Be(0);
    }